packages/filesystem/src/node/chokidar-watcher/chokidar-filesystem-watcher.ts

```diff
@@ -38,40 +44,42 @@ export class ChokidarFileSystemWatcherServer implements FileSystemWatcherServer
     protected readonly toDisposeOnFileChange = new DisposableCollection();
 
     /* Did we print the message about exhausted inotify watches yet?  */
-    protected printedENOSPCError: boolean = false;
+    protected printedENOSPCError = false;
 
-    constructor(
-        @inject(ILogger) protected readonly logger: ILogger
-    ) { }
+    constructor(protected readonly options: {
+        verbose: boolean
+    } = { verbose: false }) { }
 
     dispose(): void {
         this.toDispose.dispose();
     }
 
-    watchFileChanges(uri: string, options: WatchOptions = { ignored: [] }): Promise<number> {
+    watchFileChanges(uri: string, rawOptions?: WatchOptions): Promise<number> {
+        const options: WatchOptions = {
+            ignored: [],
+            ...rawOptions
```

Comment:
Just to be clear this rawOptions would override the ignored: [] default right?

Reply:
yes